### DISCUSSION TITLE
Trying to access array offset on value of type null

### DISCUSSION THREAD

**Discussion by resujentoezequiel** [pj8912/php-bard-api#8] [General]
Good day,

I want to integrate BARD into my system, but I encounter an error. May I know how to solve this?

![image](https://github.com/pj8912/php-bard-api/assets/43774016/89e66c3d-8a8e-44d9-b9f8-acb91b1821d1)

![image](https://github.com/pj8912/php-bard-api/assets/43774016/a84e8135-28b2-4ecd-92fe-2e75a01dcaa5)

**Comment by jarroddavis68** [comment]
try:
"content" => $parsed_answer[4][0][1][0]
